tests/functional/spec/components/dropdown.spec.js

```diff
@@ -0,0 +1,188 @@
+const helpers = require('../../helpers');
+
+describe('Component: Dropdown', function() {
+    //Mandatory
+    describe('Given I start a Mandatory Dropdown survey', function() {
+
+        var DropdownMandatoryPage = require('../../pages/components/dropdown/mandatory/dropdown-mandatory.page');
+        var DropdownSummaryPage = require('../../pages/components/dropdown/mandatory/summary.page');
+
+        before(function() {
+            return helpers.openQuestionnaire('test_dropdown_mandatory.json');
+        });
+
+        it('When I have selected a dropdown option, Then the selected option should be displayed in the summary', function() {
+            return browser
+                .selectByValue(DropdownMandatoryPage.answer(), "Rugby is better!")
+                .click(DropdownMandatoryPage.submit())
+                .getText(DropdownSummaryPage.dropdownMandatoryAnswer()).should.eventually.contain("Rugby is better!");
+        });
+    });
+
+
+    describe('Given I start a Mandatory Dropdown survey', function() {
```

Comment:
Where you have multiple of the same surveys being launched then these should be in the same describe